build/buildall

```diff
@@ -173,22 +178,23 @@ fi
 
 [[ "$MODULE" != "" ]] && MODULE_OPT="--projects $MODULE --also-make" || MODULE_OPT=""
 
+echo "Collecting Spark versions..."
 case $DIST_PROFILE in
 
   snapshotsScala213)
-    SPARK_SHIM_VERSIONS=($(versionsFromDistProfile "snapshotsScala213"))
+    SPARK_SHIM_VERSIONS=($(versionsFromReleaseProfiles "snapshots.buildvers" "scala2.13/pom.xml"))
```

Comment:
previously snapshotXXX means snapshots+non-snapshots, I think currently it should be `snap_and_no_snap` here?